frontend/test/metabase/scenarios/admin/settings/settings.cy.spec.js

```diff
@@ -91,12 +92,9 @@ describe("scenarios > admin > settings", () => {
     emailInput().should("have.value", "other.email@metabase.com");
 
     // reset the email
-    emailInput()
-      .click()
-      .clear()
-      .type("bob@metabase.com")
-      .blur();
-    cy.wait("@saveSettings");
+    cy.request("PUT", "/api/setting/admin-email", {
+      value: "bob@metabase.com",
+    });
```

Comment:
This doesn't have to do anything with the flake, but it saves a few "clicks" in the CI by using API.